kvui.py

```diff
@@ -599,6 +599,7 @@ def connect_button_action(self, button):
             self.ctx.username = None
             async_start(self.ctx.disconnect())
         else:
+            self.ctx.username = None
```

Comment:
Since `self.ctx.username = None` is in both branch of the `if self.ctx.server`, would it be better to extract it?

Reply:
didn't notice this haha, fixed